teaser/project.py

```diff
@@ -758,18 +759,98 @@ def export_aixlib(
                     )
 
         if report:
-            try:
-                from teaser.data.output.reports import model_report
-            except ImportError:
-                raise ImportError(
-                    "To create the model report, you have to install TEASER "
-                    "using the option report: pip install teaser[report] or install "
-                    "plotly manually."
-                )
-            report_path = os.path.join(path, "Resources", "ModelReport")
-            model_report.create_model_report(prj=self, path=report_path)
+            self._write_report(path)
+        return path
+
+    def export_besmod(
+        self,
+        internal_id=None,
+        examples=None,
+        path=None,
+        THydSup_nominal=None,
+        QBuiOld_flow_design=None,
+        THydSupOld_design=None,
+        custom_examples=None,
+        custom_script=None,
+        report=False
```

Comment:
Not custom in TEASER, but could you add type hints? I think it helps PyCharm depending on the docstring settings.